source/http_stream.c

```diff
@@ -192,8 +192,8 @@ static void s_http_stream_binding_finalize(napi_env env, void *finalize_data, vo
     (void)finalize_hint;
     struct http_stream_binding *binding = finalize_data;
 
-    aws_http_message_destroy(binding->request);
-    aws_http_message_destroy(binding->response);
+    aws_http_message_release(binding->request);
```

Comment:
Actually `aws_http_message_destroy ` just call `aws_http_message_release `. But I just feel it's more clear to call `aws_http_message_release `